src/Ocelot/Values/ServiceHostAndPort.cs

```diff
@@ -18,9 +18,13 @@ public ServiceHostAndPort(string downstreamHost, int downstreamPort)
     public ServiceHostAndPort(string downstreamHost, int downstreamPort, string scheme)
         : this(downstreamHost, downstreamPort) => Scheme = scheme;
 
+    public ServiceHostAndPort(string downstreamHost, int downstreamPort, string scheme, string pathPrefix)
+        : this(downstreamHost, downstreamPort, scheme) => PathPrefix = pathPrefix;
+
     public string DownstreamHost { get; }
     public int DownstreamPort { get; }
     public string Scheme { get; }
+    public string PathPrefix { get; } = string.Empty;

```

Comment:
Have you reviewed the name of the class? 😆 
This class should strictly contain information relevant to host construction: scheme, address, and port. Nothing else!

Reply:
I agree the naming of the class no longer fits. Do you have a recommendation on how to proceed? Shall I rename the class?

Reply:
F...g bot! Get out from my repo! 😠